TaxCalculator/TaxCalculation.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TaxCalculator
+{
+    class TaxCalculation
+    {
+        public void CalculateTax(Input input)

```

Comment:
Should never pass the object just for one variable value. Security issue. You are also requiring TaxCalculation class to know which method from Input to call. Unnecessary burden. Pass the variable value instead (amount).